torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -1236,6 +1236,31 @@ RegisterOperators reg2({
       },                                                                \
       aliasAnalysisFromSchema())
 
+#define DEFINE_COMPLEX_OP_WITH_TENSOR_ARG(                               \
+    type_a, type_b, actual_type_a, actual_type_b)                        \
+  Operator(                                                              \
+      "aten::Complex." #type_a "_" #type_b "(" #type_a " x," #type_b     \
+      " y) -> complex",                                                  \
+      [](Stack* stack) {                                                 \
+        actual_type_a a;                                                 \
+        actual_type_b b;                                                 \
+        pop(stack, a, b);                                                \
+        auto comp = c10::complex<double>(a.item<double>(), b);           \
```

Comment:
Do we need a check for whether the `Tensor` has only one element?

Reply:
I think item already does that check so it's not needed